src/plugins/intel_gna/src/transformations/pwl/activation_representation_configuration.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "activation_representation.hpp"
+#include "boundaries_fq_hanlder.hpp"
+#include "openvino/core/node.hpp"
+
+namespace ov {
+namespace intel_gna {
+namespace pass {
+namespace pwl {
+
```

Comment:
Although your classes and methods are short and quite self-descriptive, it would be good to provide at lease short descriptions.